README.markdown

```diff
@@ -4189,6 +4189,8 @@ to be returned when reading `ngx.header.Foo`.
 
 Note that `ngx.header` is not a normal Lua table and as such, it is not possible to iterate through it using the Lua `ipairs` function.
 
+Notice: both the `HEADER` and the `VALUE` should not contains `\r` or `\n`, otherwise, the characters after `\r` or `\n` will be truncated, include the `\r` or `\n` themself.
```

Comment:
Grammar: "include" => "including".

"should not contains" => "should not contain".